apisix/plugins/hmac-auth.lua

```diff
@@ -328,7 +312,7 @@ local function get_params(ctx)
     local date_key = DATE_KEY
     local signed_headers_key = SIGNED_HEADERS_KEY
 
-    if try_attr(local_conf, "plugin_attr", "hmac-auth") then
+    if core.table.try_read_attr(local_conf, "plugin_attr", "hmac-auth") then
         local attr = local_conf.plugin_attr["hmac-auth"]
```

Comment:
`try_read_attr` already gets what we need, we don't need to read it again.

Reply:
OK. I had fixed.